doc/content/gateways/lora-basics-station/troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Troubleshooting LBS"
+description: ""
+---
+
+This section contains help for common problems you may encounter when using {{% lbs %}} to connect to {{% tts %}}.
+
+<!--more-->
+
+## What is my server address?
+
+This is the address you use to access {{% tts %}}. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
+
+CUPS uses the URI: `https://<server-address>:443`
+
+LNS uses the URI: `wss://<server-address>:8887`
+
+## How do I find the CA Trust?
+
+Some device manufacturers include [common CA's](https://www.ccadb.org/) in device firmware, and these devices should connect automatically to {{% tts %}}.
+
+If your device does not contain common CA's, and you are using Let's Encrypt to secure your domain, you may download the LEX3 Trust file [here](https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt). Save the contents of the file as `cert.pem` and upload it as the Server Certificate on your gateway when connecting to {{% lbs %}}.
+
+If you are using self signed certificates, you are your own Trust. You may generate a Root Certificate from your private key using openssl:
+
+```bash
+$ openssl req -x509 -new -nodes -key <rootCA.key> -sha256 -days 1024  -out <rootCA.pem>
+```
+
+## How do I use an API Key?
+
+This varies from device to device. If your device allows you to upload a `.key` file, copy your gateway API Key in to a `gateway.key` file and upload it. Otherwise, see your manufacturers guidelines for additional information.
```

Comment:
It's not just the API Key but this should be a valid [HTTP Header](https://doc.sm.tc/station/authmodes.html#tls-server-authentication-and-client-token) with the correct line endings
So
```
Authorization: <API Key> \r\n
```

It's better we test this. Ask me if you need help with this.

Reply:
I don't have an LBS gateway here, wanna try it?